src/pages/_nav_bar.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { animated, useSprings } from "@react-spring/web";
+import { MenuAlt2Icon, ArrowRightIcon } from "@heroicons/react/outline";
+
+function NavBarComponent({ customMouseEnter, customMouseLeave }: any) {
+  const bg_colors = ["bg-amber-400", "bg-red-400", "bg-blue-400"];
+  const [_springs, _apis] = useSprings(bg_colors.length + 1, (index) => ({
```

Comment:
Consider using the `useEffect` hook to animate the dropdown menu instead of the `useSprings` hook is more performant than the `useSprings`